datalad_container/containers_run.py

```diff
@@ -87,6 +87,8 @@ def __call__(cmd, container_name=None, dataset=None,
                     fullcmd.extend(cmd)
                 elif c == '{img}':
                     fullcmd.append(image_path)
+                elif c == '{pwd}':
```

Comment:
This wouldn't work in the common case of `--binding src:dest`.  When thinking about the `{input}/{output}` fields for `datalad run`, I've wondered whether we should switch to just always running a string  command and `shell=True`, creating the string from the list of args if necessary.  We could then always use `str.format` to expand the fields.

Reply:
Yeah, I am gravitating more and more in this direction too. It would simplify things quite a bit. Can you this?

Reply:
> I am gravitating more and more in this direction too. It would simplify things quite a bit. Can you this?

Sure, I'll open a PR for this today.